obspy/io/nordic/ellipse.py

```diff
@@ -0,0 +1,329 @@
+"""
+Routines for error ellipses
+ - Calculate an ellipse from covariance matrix
+ - See if a point is inside an ellipse
+ - Calculate the angle subtended by an ellipse (for back-azimuth uncert)
+ - Plot an ellipse
+
+Still missing:
+ - ellipsoids (3D ellipses)
+ - to_uncert()  (convert ellipse to covariance matrice, needed to
+                 convert QuakeML to nordic, for example)
+"""
+
+import numpy as np
+import matplotlib.pyplot as plt
+import copy
+
+eps = np.finfo(float).eps
```

Comment:
Can you call this `EPS` for global variables.  Is there a good reason for this being global?

Reply:
I don't know why I made this a global, maybe I thought I'd use it more than once.

I now just call it directly int the function